src/pages/landing/landingPage.css

```diff
@@ -0,0 +1,110 @@
+body {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    overflow-x: hidden;
+  }
+  
+  :root {
+    --primary-color: #ff6f61;
+    --black-text: #1e1c1c;
+    --facebook-blue: #1877f2;
+    --hero-bg: #fffff0;
+    --login-bg: #ffffff;
+    --page-containers: #e5e5e5;
+  }
+  
+  a {
+    color: var(--black-text);
+  }
+  
+  .primary-highlight {
+    color: var(--primary-color);
+  }
+  
+  .nav-bar {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 2rem 2rem;
+  }
+  
+  .left-navbar {
+    cursor: pointer;
+    font-size: 1rem;
+  }
+  
+  .right-navbar {
+    display: flex;
+    gap: 2.5rem;
+    align-items: center;
+  }
+  
+  .landing-content {
+    display: flex;
+    background-color: var(--hero-bg);
+    align-items: center;
+    justify-content: space-around;
+    height: 38.5rem;
+  }
+  
+  .landing-image img {
+    width: 31.875rem;
+    height: 23.75rem;
+  }
+  
+  .landing-text {
+    font-size: 1.875rem;
+    width: 40rem;
+  }
+  
+  .btn-getstarted {
+    margin-top: 2rem;
+    padding: 1rem 5rem;
+    background-color: var(--primary-color);
+    border-radius: 0.5rem;
+    cursor: pointer;
+    border: none;
+    font-size: 1rem;
+  }
+  
+  .flex_c {
+    display: flex;
+    flex-direction: column;
+  }
+  
+  .flex_r {
+    display: flex;
+  }
+  
+  .padding-bottom {
+    padding-bottom: 2rem;
+  }
+  
+  .bg-gray {
+    background-color: #908988;
```

Comment:
Use can use a var here instead of directly using a color name.

Reply:
Noted! Thanks for the feedback